### PR TITLE
update support for accelerated_field of kvstore

### PR DESCRIPTION
**Pull request by akaila-splunk**
updated accelerated_field methods and added test cases for it. (issue #497 for ref)

### CODE REVIEW

File: splunklib/client.py [hunk 1]
```diff
@@ -3740,18 +3747,20 @@ def data(self):
         """
         return KVStoreCollectionData(self)
 
-    def update_index(self, name, value):
```
Comment (fantavlik) [inline splunklib/client.py:3743]: I think we want to add support for accelerated field indexes without removing current index support

**Review by fantavlik — CHANGES_REQUESTED** [review]
~Recommend adding accelerated fields support without removing current functionality if possible~

Since this functionality was deprecated in 7.x I'm good with moving forward.

File: splunklib/client.py (hunk 1 above)
Comment (akaila-splunk) [reply splunklib/client.py:3743]: In Splunk Enterprise v6.4 archive docs we found that for kvstore `index.<indexName>` property exist but in later version it was updated to `accelerated_field.<field_name>`, so due to this change we updated the same method. PFA docs screenshots for reference, let us know if we still want existing index support.

Splunk v6.4 docs:
splunkv6 4_doc

Splunk v8.2 docs:
splunkv8 2_doc

**Review by vishwambhar — APPROVED** [review]

File: splunklib/client.py (hunk 1 above)
Comment (fantavlik) [reply splunklib/client.py:3743]: OK great thanks for clarifying, in that case the changes look good to me 👍

**Pull request merged**